recipe/meta.yaml

```diff
@@ -33,8 +33,7 @@ requirements:
     - python-dateutil
     - myproxyclient
```

Comment:
`esgf-pyclient` and `myproxyclient` should be excluded for python 3 builds since the latter isn't compatible.